frontend/src/components/UserInfo/UserAvatarWithInfo.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Redirect } from 'react-router';
+import {withStyles} from '@material-ui/core/styles';
+import UserInfo from "./UserInfo";
+import UserAvatar from "./UserAvatar";
+
+const styles = {
+    forDiv: {
+        flexGrow: 1,
+        display: "flex",
+        justifyContent: "center",
+    },
+};
+
+class UserAvatarWithInfo extends React.Component {
+
+    state = {
+        userInfo: [],
+        isForbidden: false,
+    };
+
+    componentDidMount() {
+        fetch('http://localhost:6543/profile')
+            .then(response => response.json())
+            .then(data => this.setState({userInfo: data.data[0], isForbidden: data.error.toLowerCase().search("forbidden") === -1 ? false: true}))
+            .catch(err=>console.log(err))
+    }
+
+    render() {
+        const {classes} = this.props;
+        const {userInfo, isForbidden} = this.state;
+        console.log(isForbidden);
+        if (isForbidden){
```

Comment:
as i understood it is temporary, right?

Reply:
all logs I removed (but didn't commit yet) - I forgot remove them before commit.

Reply:
i mean isForbidden cheeking, not console.log()

Reply:
If isForbidden is "true" - I return Redirection, else I return false. It influence in "UserAvatarWithInfo.js" in render() method